app/views/tag_groups/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title_bar do %>
+  <%= link_to t('back'), root_path, class: "btn btn-primary btn-sm navbar-btn" %>
+<% end %>
+
+<div class="row">
+  <div class="col-xs-12 col-sm-5 col-sm-offset-2">
+    <div class="page-header">
+      <h4 class="page-header-title darkgrey-2">
+        <%= t('projects.edit tag_group') %>
+      </h4>
+    </div>
+  </div>
+  <div class="col-xs-12 col-sm-5 col-sm-offset-2">
+    <div class="panel card">
+      <div class="panel-body"
+      data-current_color=<%= @tag_group[:bg_color] %> >
```

Comment:
align this with class attribute. 

Optionally you can rename this to `data-current-color`.